proto/regen/ecocredit/basket/v1/query.proto

```diff
@@ -46,7 +47,7 @@ message QueryBasketRequest {
 // QueryBasketResponse is the Query/Basket response type.
 message QueryBasketResponse {
   // basket is the queried basket.
-  Basket basket = 1;
+  BasketInfo basket = 1;
```

Comment:
We are not bumping the basket proto version in this release so we may need to deprecate and add rather than updating the existing messages and fields:

https://github.com/cosmos/cosmos-sdk/blob/main/docs/architecture/adr-044-protobuf-updates-guidelines.md

We started using `Since Revision 1` for fields we are adding rather than what would be `Since: v4.0` based on the guidelines linked above. Let's discuss this on our call tomorrow.

Reply:
I'd also go with the former approach. If possible, the deprecated field should be populated too, to offer maximum backwards compatibiliy.